composer.json

```diff
@@ -16,19 +16,19 @@
             "name":"Michael Bodnarchuk"
         }
     ],
-    "minimum-stability": "RC",
+    "minimum-stability": "dev",
     "require": {
-        "php": ">=7.4.0 <=8.1 | ~8.1.0",
-        "codeception/lib-innerbrowser": "^1.0",
-        "codeception/codeception": "^4.0"
+        "php": ">=8.1.0",
+        "codeception/codeception": "dev-5.0-interfaces as 5.0.0",
+        "codeception/lib-innerbrowser": "^3.0"
     },
     "require-dev": {
         "yiisoft/yii2": "dev-master",
         "yiisoft/yii2-app-advanced": "dev-master",
         "codeception/verify": "<2",
```

Comment:
It's not related to Codeception 5 compatibility at all, but since this will dump a major version we could take advantage of upgrading verify to version 2.2 https://github.com/Codeception/Verify/blob/master/UPGRADE.md

Reply:
```
Problem 2
    - Root composer.json requires codeception/codeception ^5.0.0-alpha2 -> satisfiable by codeception/codeception[5.0.0-alpha2, 5.0.x-dev].
    - codeception/verify 2.2.0 requires phpunit/phpunit ^9.3 -> satisfiable by phpunit/phpunit[9.3.0, ..., 9.5.x-dev].
    - You can only install one version of a package, so only one of these can be installed: phpunit/phpunit[dev-master, 7.5.0, ..., 7.5.20, 8.0.0, ..., 8.5.x-dev, 9.0.0, ..., 9.5.x-dev].
    - codeception/codeception[5.0.0-alpha2, ..., 5.0.x-dev] require phpunit/phpunit dev-master as 9.5.99 -> satisfiable by phpunit/phpunit[dev-master].
    - Root composer.json requires codeception/verify ^2.2 -> satisfiable by codeception/verify[2.2.0].

```

https://github.com/Codeception/Verify/pull/53

Reply:
@developedsoftware you're ready to go now!

Reply:
I dont think I can upgrade codeception/verify to 2.2

It breaks the tests in yii2 which is able to run on php 5!

https://github.com/yiisoft/yii2-app-advanced/pull/519

Propose dropping back down to "codeception/verify": "<2"

Thoughts?

Reply:
gh doesn't notify me if you edit a comment, make sure to tag me when you do.

Reply:
Ok I think this PR is good to go. But I agree verify 2.2 is worth upgrading to if yii2 is able to drop php5 and require a min version of php7

Reply:
Ok I think this PR is good to go. But I agree verify 2.2 is worth upgrading to if yii2 is able to drop php5 and require a min version of php7

Reply:
> It's not related to Codeception 5 compatibility at all, but since this will dump a major version we could take advantage of upgrading verify to version 2.2 https://github.com/Codeception/Verify/blob/master/UPGRADE.md

When pull request https://github.com/yiisoft/yii2-app-advanced/pull/521 is merged we should be able to bump to codeception/verify 2.2

Reply:
Tests now pass.
<img width="851" alt="Screenshot 2022-02-25 at 09 11 01" src="https://user-images.githubusercontent.com/19648131/155688012-28e7dcb1-06d9-4d33-8f94-784d4dbdcbe1.png">